_posts/2020-02-14-where-are-we-at-with-waku.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+name:  "Where Are We at with Waku?"
+title:  "Where Are We at with Waku?"
+date:   2020-02-10 12:00:00 +0800
+author: oskarth
+published: true
+permalink: /where-are-we-at-with-waku
+categories: research
+summary: A research log. Where are we at with Waku?
+image: /assets/img/waku_infrastructure_sky.jpg
+discuss: https://forum.vac.dev/t/TODOFIXME
+---
+
+TODO: Picture for bottleneck or Waku
+TODO: Fix discuss link
+TODO: Fix image credit
+
+# Where we are at with Waku?
+
+Waku is our fork of Whisper where we address the shortcomings of Whisper in an iterative manner. We've seen a in [previous post](https://vac.dev/fixing-whisper-with-waku) that Whisper doesn't scale, and why. Here's an update on where we are at with Waku since then.
+
+## Current state
+
+**Specs:**
+
+We've cut the spec up into three components. These are:
+
+- Waku (main spec), currently in [version 0.3.0](https://specs.vac.dev/waku/waku.html)
+- Waku envelope data field, currently in [version 0.1.0](https://specs.vac.dev/waku/envelope-data-format.html)
+- Waku mailserver, currently in [version 0.2.0](https://specs.vac.dev/waku/mailserver.html)
+
+**Clients:**
+
+There are currently two clients that implement Waku, these are [Nimbus](https://github.com/status-im/nimbus/tree/master/waku) in Nim and [status-go](https://github.com/status-im/status-go) in Go.
+
+At the time of writing the Nimbus client implements the spec fully, but lacks mail server/client, rate limiting and confirmations capability. The status-go client implements everything except bridging mode, which is currently a work in progress.
+
+For more details, see [implementation matrix](https://specs.vac.dev/waku/waku.html#appendix-b-implementation-notes).
+
+In terms of end user applications, work is currently in progress to integrate it into the [Status core app](https://github.com/status-im/status-react/pull/9949) and is expected to be released in their upcoming 1.1 release.
+
+TODO: Fact check this with Adam and Kim
+
+**Simulation:**
+
+We've got a [simulation](https://github.com/status-im/nimbus/tree/master/waku#testing-waku-protocol) in the Nimbus client that verifies - or rather, fails to falsify - the scalability model described in an [earlier post](https://vac.dev/fixing-whisper-with-waku). More on this below.
+
+## How many users does Waku support?
+
+This is our current understanding of how many users a network running the Waku protocol can support. Specifically in the context of the Status chat app, since that's the most immediate consumer of Waku. It should generalize fairly well to most deployments, but YMMW.
```

Comment:
what is YMMW?

Reply:
your milage might vary (typo)

removed